packages/gatsby/src/schema/infer-graphql-type.js

```diff
@@ -400,3 +407,7 @@ export function inferObjectStructureFromNodes({
 
   return inferredFields
 }
+
+export function inferObjectStructureFromNodes(options: inferTypeOptions) {
```

Comment:
this is prevent passing in `exampleValue` from the outside, it was always intended to be internal but existed as an arg for recursion's sake